src/ONYXKEYS.js

```diff
@@ -101,6 +101,7 @@ export default {
         REPORTS_WITH_DRAFT: 'reportWithDraft_',
         REPORT_IS_COMPOSER_FULL_SIZE: 'reportIsComposerFullSize_',
         IS_LOADING_INITIAL_REPORT_ACTIONS: 'isLoadingInitialReportActions_',
+        IS_LOADING_REPORT_ACTIONS: 'isLoadingReportActions_',
```

Comment:
What's the difference between this collection, the one above it, and the one below it? This feels like it's getting a little bit out of hand.

Reply:
Why are we using collections for these things at all?

Reply:
I think we can remove the `IS_LOADING_INITIAL_REPORT_ACTIONS` one. It seems like nothing is subscribing to this key. So that just leaves the others...

But I _think_ we are using collections because: 

- an individual report could be "loading" or "loading more" while not in view and we don't want to incorrectly show a report as loading when it's not or not loading when it is
- there wasn't any other alternative proposed

Any specific concern with this approach? 

Thinking on what we should we do as an alternative... One idea is to possibly use the `report` object to store a `isLoadingActions` and `isLoadingMoreActions`. The report object is a little bloated already with all sorts of random stuff, but maybe it's better than what we have here.

Reply:
> One idea is to possibly use the report object to store a isLoadingActions and isLoadingMoreActions.

Yeah, this is basically what I would propose otherwise. It sounds like that's the best practice considering pattern B.

My concern is that I think this leads to an anti-pattern. It becomes a bunch of keys holding a single boolean value, when the boolean value could be a property on the normal report collection. If a new engineer looks at this, and sees it's OK, then what are the next collections with single values? Collections for just report names? You can use this argument to say that instead of one collection with `n` reports, then it's OK to have `n` collections for `n` reports (one collection for each report property).

Unless there is a real advantage to having these kinds of simple value collections (I am just not seeing it), I think we should avoid it and stick to our best practices. (not trying to blame anyone!)

Reply:
Wanna move this to Slack to discuss? After, we can put it down in the style guide / fix anything that is not following the best practice.

Reply:
I will remove the `isLoadingInitialReportActions_`

Reply:
Ok seems like we had a good discussion [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1660150371779569) and are adding this to the style guide. 

So, @sketchydroide we just need to update these so that the boolean properties are on the report instead of collection keys.

Reply:
I think I have done this for the `isLoadingReportActions`
Should I do it for the IS_LOADING_MORE_REPORT_ACTIONS as well in this PR as well, it's from the previous PR for `ReadOldestAction`

Reply:
I'm not 100% I did that correctly, but I think so, did I miss something, it feels odd that `isLoadingReportActions` is not always defined...

Reply:
Looks correct to me. 

>it feels odd that isLoadingReportActions is not always defined...

This is JavaScript which means 99% of all things are possibly `undefined`, but hey at least it's a boolean!